app/views/videos/backup.html.erb

```diff
@@ -0,0 +1,58 @@
+<div class="main-container">
+  <div class="container">
```

Comment:
What is this backup for?

Reply:
oh i think can take away, prima guided me on a new way of doing the index page. He created a backup to keep what i done before

Reply:
ok, then can keep it.